scripts/reindex.py

```diff
@@ -96,8 +106,24 @@ def main(argv: List[str]):
 
     configure_script_logging(logger)
 
+    sources = set(args.sources.split(','))
```

Comment:
I know I suggested comma as the separator but I would like to try shell-style word splitting on consecutive whitespace instead. It's more consistent with using a shell-like mechanism for globbing and it would allow us to use `nargs='+'` again.